Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.Action.ps1

```diff
@@ -3,6 +3,8 @@
     [string] $baseFolder,
     [Parameter(HelpMessage = "The maximum depth to build the dependency tree", Mandatory = $false)]
     [int] $maxBuildDepth = 0,
+    [Parameter(HelpMessage = "Specifies whether to publish artifacts for skipped projects", Mandatory = $false)]
```

Comment:
It is confusing to have `$publishSkippedProjectArtifacts` here, as this action doesn't publish anything.